config/locales/en.yml

```diff
@@ -62,7 +63,7 @@ en:
   phone_number: 'Cell phone number'
   address: 'Address'
   city: 'City'
-  state: 'State'
+  state: 'state'
```

Comment:
Yeah. I vacillate on capitalization in localization files, not sure what best practices are.